README.md

```diff
@@ -1,17 +1,68 @@
 # FastQC-nf bioinformatics workflow
 
+ - [Description](#description)
+ - [User Guide](#user-guide)
+ - [Additional notes](#additional-notes)
+ - [Acknowledgements](#acknowledgements)
+
 ## Description
-FastQC-nf is a pipeline for quality control in both illumina short reads and Pacbio long reads. It's written in nextflow. 
 
+<p align="center">
+:wrench: This pipeline is currently under development :wrench:
+</p>
+
+FastQC-nf is a Nextflow workflow for evaluating the quality of high-throughput sequencing reads. It employs [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) to assess the quality of individual fastq files. FastQC metrics are then aggregated into a single HTML report using [MultiQC](https://multiqc.info/). This pipeline accepts both Illumina short read and PacBio long read datasets. 
+
```

Comment:
Changed the description to focus on the pipeline we're building rather than nextflow and clarify it is under development.